docs/en/writing-running-appium/server-args.md

```diff
@@ -93,3 +93,5 @@ All flags are optional, but some are required in conjunction with certain others
 |`--localizable-strings-dir`|en.lproj|[DEPRECATED] - (IOS-only) the relative path of the dir where Localizable.strings file resides |`--localizable-strings-dir en.lproj`|
 |`--show-ios-log`|false|[DEPRECATED] - (IOS-only) if set, the iOS system log will be written to the console||
 |`--relaxed-security`|false|Disable additional security checks, so it is possible to use some advanced features, provided by drivers supporting this option. Only enable it if all the clients are in the trusted network and it is not the case if a client could potentially break out of the session sandbox.||
```

Comment:
it makes sense to mention that both allow/deny-insecure have priority over that option

Reply:
But --allow-insecure does not have priority over --relaxed-security. If they are both used then --allow-insecure is doing nothing.